src/cdk/datetime/datetime.md

```diff
@@ -0,0 +1,111 @@
+The `datetime` package offers building blocks for custom date and time-based components.
+
+### `DateAdapter`
+The CDK `DateAdapter` facilitates use of any date data structure for date and time-based components and gives a
+foundation for more concrete `DateAdapter` implementations.
+
+#### Using a `DateAdapter`
+A `DateAdapter` provides a uniform way to deal with different date representations. Directives that deal with dates use
```

Comment:
I missed this earlier, but the text should be wrapped at 100 columns for the whole file